### ISSUE TITLE
Admin Search Suggestions Toggle Not Working

### ISSUE THREAD

**Issue by Stoyvo** [magento/magento2#21372]
### Preconditions (*)
1. Magento 2.2.5

### Steps to reproduce (*)
1. In admin, navigate to Store -> Configuration -> Catalog -> Catalog Search
2. Set "Search Recommendations" and "Search Suggestions" both to "No"
3. On frontend, type a search term and press enter
4. On frontend, start typing the same search term

### Expected result (*)
1. No search query should run and no results

### Actual result (*)
1. Search still initiated at /search/ajax/suggest/?q=
2. Switching this toggle should disable search suggestions

**Comment by magento-engcom-team** [comment]
Hi @Stoyvo. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@Stoyvo do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by Stoyvo** [comment]
@magento-engcom-team 
Yes.

**Comment by Usuario001** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Usuario001. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Usuario001, here is your [Magento instance](https://i-21372-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21372-2-3-develop.instances.magento-community.engineering/admin](https://i-21372-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by Stoyvo** [comment]
@magento-engcom-team give me 2.3 instance

**Comment by Stoyvo** [comment]
Can confirm this appears on the instance @Usuario001 requested

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by Stoyvo** [comment]
FWIW, It looks like the handy tool to build these instances fails to build elasticsearch and all attempts to search will render 503 error page with generated report number.

To circumvent this, change search engine to MYSQL in the new instances admin

**Comment by shikhamis11** [comment]
@magento-engcom-team give me 2.2-develop instance

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11. Thank you for your request. I'm working on Magento 2.2-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @shikhamis11, here is your [Magento instance](https://i-21372-2-2-develop.instances.magento-community.engineering).
Admin access: [https://i-21372-2-2-develop.instances.magento-community.engineering/admin](https://i-21372-2-2-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @Nazar65. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98692` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by Tjitse-E** [comment]
This issue still exists on 2.3-dev (and Magento 2.3.3).

**Comment by BrentRobert** [comment]
This issue has already been fixed by https://github.com/magento/magento2/commit/e567fd6c343cd383bc58b8b6b8b90b45844ded7d

**Comment by engcom-Charlie** [comment]
Hello @Stoyvo the issue is already fixed in the latest 2.4-develop.
Please feel free to comment, reopen or create a new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
if you are still facing this issue on the latest `2.4-develop` branch. Thank you for your collaboration.

Thanks for your report!

**Comment by magento-engcom-team** [comment]
Hi @Stoyvo, @engcom-Charlie. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-24395` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
